extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java

```diff
@@ -470,6 +521,17 @@ public void accept(Path path) {
             reflectiveClasses.accept(new ReflectiveClassBuildItem(false, false, i.getListenerClass()));
             template.registerListener(deployment, context.classProxy(i.getListenerClass()), bc.getValue());
         }
+
+        for (ServletContainerInitializerBuildItem i : servletContainerInitializerBuildItems) {
+            Set<Class<?>> handlesTypes = new HashSet<>();
+            for (String e : i.handlesTypes) {
```

Comment:
Can't say I'm a big fan of you using single letter variables, every time you have a for loop. It's quite hard to know what they are referring to. (This is a general comment for the whole patch)

It's really just cosmetic but it's not the first time I have a hard time following this sort of patterns in the code so I thought I might as well raise the issue.